test/resources/common.resource

```diff
@@ -20,7 +20,7 @@ Setup Suite With Namespace
     Login MicroShift Host
     Setup Kubeconfig
     ${ns}=    Create Unique Namespace
-    Set Suite Variable    \${NAMESPACE}    ${ns}
+    VAR    ${NAMESPACE}=    ${ns}    scope=SUITE
```

Comment:
Looking at [the release notes](https://github.com/robotframework/robotframework/blob/master/doc/releasenotes/rf-7.0.rst#native-var-syntax) I think we don't need the `=`
```suggestion
    VAR    ${NAMESPACE}    ${ns}    scope=SUITE
```